README.md

```diff
@@ -30,7 +30,7 @@ The only dependency is the [Gremlin.Net driver](https://www.nuget.org/packages/G
 - Using Powershell Package Manager:
 
     ```
-    Install-Package Gremlin.Net -Version 3.4.0-rc2
+    Install-Package Gremlin.Net -Version 3.4.6
     ```
 
```

Comment:
Could we also call out in the instructions that they need to specify their credentials as environment variables now?